styles/gauge.mapcss

```diff
@@ -0,0 +1,1165 @@
+{
+  z-index: 16;
+  color: #4000FF;
+}
+
+way[gauge=1050].tracks,
+way[gauge>1050][gauge<1066].tracks
+{
+  z-index: 17;
+  color: #0000FF;
+}
+
+way[gauge=1066].tracks,
+way[gauge>1066][gauge<1100].tracks
+{
+  z-index: 18;
+  color: #0000E0;
+}
+
+way[gauge=1100].tracks,
+way[gauge>1100][gauge<1200].tracks
+{
+  z-index: 19;
+  color: #0000C0;
+}
+
+way[gauge=1200].tracks,
+way[gauge>1200][gauge<1372].tracks
+{
+  z-index: 20;
+  color: #0000A0;
+}
+
+way[gauge=1372].tracks,
+way[gauge>1372][gauge<1422].tracks
+{
+  z-index: 21;
+  color: #000080;
+}
+
+way[gauge=1422].tracks,
+way[gauge>1422][gauge<1432].tracks
+{
+  z-index: 22;
+  color: #000060;
+}
+
+way[gauge=1432].tracks,
+way[gauge>1432][gauge<1435].tracks
+{
+  z-index: 23;
+  color: #000040;
+}
+
+way[gauge=1435].tracks,
+way[gauge>1435][gauge<1440].tracks
+{
+  z-index: 0;
+  color: #000000;
+}
+
+way[gauge=1440].tracks,
+way[gauge>1440][gauge<1445].tracks
+{
+  z-index: 25;
+  color: #400000;
+}
+
+way[gauge=1445].tracks,
+way[gauge>1445][gauge<1450].tracks
+{
+  z-index: 26;
+  color: #600000;
+}
+
+way[gauge=1450].tracks,
+way[gauge>1450][gauge<1458].tracks
+{
+  z-index: 27;
+  color: #700000;
+}
+
+way[gauge=1458].tracks,
+way[gauge>1458][gauge<1495].tracks
+{
+  z-index: 28;
+  color: #800000;
+}
+
+way[gauge=1495].tracks,
+way[gauge>1495][gauge<1520].tracks
+{
+  z-index: 29;
+  color: #A00000;
+}
+
+way[gauge=1520].tracks,
+way[gauge>1520][gauge<1522].tracks
+{
+  z-index: 30;
+  color: #C00000;
+}
+
+way[gauge=1522].tracks,
+way[gauge>1522][gauge<1524].tracks
+{
+  z-index: 31;
+  color: #E00000;
+}
+
+way[gauge=1524].tracks,
+way[gauge>1524][gauge<1581].tracks
+{
+  z-index: 32;
+  color: #FF0000;
+}
+
+way[gauge=1581].tracks,
+way[gauge>1581][gauge<1588].tracks
+{
+  z-index: 33;
+  color: #FF6000;
+}
+
+way[gauge=1588].tracks,
+way[gauge>1588][gauge<1600].tracks
+{
+  z-index: 34;
+  color: #FF8000;
+}
+
+way[gauge=1600].tracks,
+way[gauge>1600][gauge<1668].tracks
+{
+  z-index: 35;
+  color: #FFA000;
+}
+
+way[gauge=1668].tracks,
+way[gauge>1668][gauge<1676].tracks
+{
+  z-index: 36;
+  color: #FFC000;
+}
+
+way[gauge=1676].tracks,
+way[gauge>1676][gauge<1700].tracks
+{
+  z-index: 37;
+  color: #FFE000;
+}
+
+way[gauge=1700].tracks,
+way[gauge>1700][gauge<1800].tracks
+{
+  z-index: 38;
+  color: #FFFF00;
+}
+
+way[gauge=1800].tracks,
+way[gauge>1800][gauge<1880].tracks
+{
+  z-index: 39;
+  color: #E0FF00;
+}
+
+way[gauge=1880].tracks,
+way[gauge>1880][gauge<2000].tracks
+{
+  z-index: 40;
+  color: #C0FF00;
+}
+
+way[gauge=2000].tracks,
+way[gauge>2000][gauge<3000].tracks
+{
+  z-index: 41;
+  color: #A0FF00;
+}
+
+/* miniature tracks */
+
+way[gauge="2.5inch"].tracks,
+way[gauge="2_1/2_inch"].tracks,
+way[gauge="2½\""].tracks,
+way[gauge="2.5\""].tracks,
+way[gauge="0'2.5\""].tracks,
+way[gauge=63].tracks,
+way[gauge>63][gauge<88].tracks
+{
+  z-index: 42;
+  color: #006060;
+}
+
+way[gauge="3.5inch"].tracks,
+way[gauge="3_1/2_inch"].tracks,

```

Comment:
Underscores and slashes make our MapCSS compiler (run `cd styles && make`) freak out. JOSM would work if we did not use `@import` which requires preprocessing using `awk`. Please get rid of them.

To be honest, I don't understand why you want to support any other way to provide gauge than the pure number in millimeter. They are not frequently used and the map styles should not support broken tagging.

Reply:
Done. I removed the inch stuff and other broken tagging. Since it would have to be millimeter anyway, we don't need that here.